src/com/goide/project/GoLibrariesService.java

```diff
@@ -38,7 +38,7 @@
 import java.util.Set;
 
 public abstract class GoLibrariesService<T extends GoLibrariesState> extends SimpleModificationTracker implements PersistentStateComponent<T> {
-  public static final Topic<LibrariesListener> LIBRARIES_TOPIC = new Topic<LibrariesListener>("libraries changes", LibrariesListener.class);
+  public static final Topic<LibrariesListener> LIBRARIES_TOPIC = new Topic<LibrariesListener>("Go packages changes", LibrariesListener.class);
```

Comment:
Did you meant the name of the constant itself, from `LIBRARIES_TOPIC` to `GOPATH_TOPIC` (I think it's better suited that `Go Packages` in this context).